examples/chunker/10_clean_data.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2021-2024 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+import json
+import logging
+import pathlib
+import re
+
+from config import CLEANED_DOCUMENTS_JSON_FILE_LOCATION
+from config import DOCUMENTS_JSON_FILE_LOCATION
+from vdk.api.job_input import IJobInput
+
+log = logging.getLogger(__name__)
+
+
+def clean_text(text):
```

Comment:
why do you think this is needed? I really think this is a bad idea. I left a similar comment on a differnt PR https://github.com/vmware/versatile-data-kit/pull/3117#discussion_r1491023553. 
Please take a look.

Reply:
Well, as I also came to a similar conclusion last week, we created #3089 to come up with a better approach as the current one is not great - I would say 100% lemmatization and stop-words removal should be removed. About the punctuation and the lower case I'm not sure. I don't think it would necessarily damage the model but we might get rid of them as well.

Reply:
Done - removed the cleaning part.